### PR TITLE
Use bole for logging

### PR DESCRIPTION
**Pull request by aknuds1**
Use the [bole logger](https://github.com/rvagg/bole).

### CODE REVIEW

**Comment by yoshuawuyts** [comment]
Looking great! - just a lil comment is all :sparkles: - would be super keen to merge this :gring:

File: test/index.js [hunk 1]
```diff
@@ -106,8 +106,8 @@ test('start', function (t) {
         }
 
         const actual = data.toString().split('\n')[0]
-        const expected = `Started bankai for fixture.js on http://localhost:${port}`
-        t.equal(actual, expected, 'start logs success')
+        const expected = /^\[\d+\] info {2}Started bankai for fixture.js on http:\/\/localhost:1024 \(bankai.start\)$/
```
Comment (aknuds1) [inline test/index.js]: @yoshuawuyts Ah, oversight is all :/

**Comment by yoshuawuyts** [comment]
test if this does work

**Comment by yoshuawuyts** [comment]
seem it took a while for my emails to arrive 😅

**Pull request merged**